databricks/koalas/internal.py

```diff
@@ -622,25 +622,27 @@ def offset(id):
     def _column_index_to_name(self) -> Dict[Tuple[str, ...], str]:
         return dict(zip(self.column_index, self.data_columns))
 
-    def column_name_for(self, column_name_or_index: Union[str, Tuple[str, ...]]) -> str:
+    def column_name_for(self, column_index_or_index_column: Union[str, Tuple[str, ...]]) -> str:
```

Comment:
BTW, is it a proper term used in pandas? Column index and index column .... seems very confusing :-)..

Reply:
Agree! 😄 
hm, should we use "label" instead?

https://pandas.pydata.org/pandas-docs/stable/reference/frame.html

> DataFrame.index | The index (row labels) of the DataFrame.
> DataFrame.columns | The column labels of the DataFrame.

Reply:
sounds good!

Reply:
maybe let me change it in a separate PR?
I guess we should rename others as well, like `column_index` -> `column_labels`?

Reply:
no problem